stories/button/index.js

```diff
@@ -91,6 +91,20 @@ stories
         </Button>
       </p>
     </Container>
+  ).add('Basic with Icon', () =>
+    <Container text style={{ padding: "2rem" }}>
```

Comment:
Added aa basic button to story, no changes were made to Basic icon. I thought that I would have to change the styling for it, but I was supposed to use a tertiary button in the first place. (Yey for easy fixes)